contracts/interfaces/IInterchainTokenService.sol

```diff
@@ -170,19 +172,28 @@ interface IInterchainTokenService is
     function interchainTokenId(address operator_, bytes32 salt) external view returns (bytes32 tokenId);
 
     /**
-     * @notice Deploys a custom token manager contract on a remote chain.
+     * @notice Registers metadata for a token on the ITS Hub. This metadata is used for scaling linked tokens.
+     * @param tokenAddress The address of the token.
+     * @param gasValue The cross-chain gas value for sending the registration message to ITS Hub.
+     */
+    function registerTokenMetadata(address tokenAddress, uint256 gasValue) external payable;
+
+    /**
+     * @notice Links a source token to a destination token on a remote chain.
      * @param salt The salt used for token manager deployment.
      * @param destinationChain The name of the destination chain.
+     * @param destinationTokenAddress The address of the token on the destination chain.
      * @param tokenManagerType The type of token manager. Cannot be NATIVE_INTERCHAIN_TOKEN.
-     * @param params The deployment parameters.
+     * @param linkParams The link parameters.
```

Comment:
mention that note on operator being linkParams for EVM case

Reply:
done